api/urls.py

```diff
@@ -61,4 +62,7 @@
     path('add-feedback', feedback_views.add_feedback, name="add-feedback"),
     path('get-all-user-feedback', feedback_views.get_all_user_feedback, name="get-all-user-feedback"),
     path('get-feedback/<int:feedback_id>', feedback_views.get_feedback, name="get-feedback"),
+
+    # Currency Conversion
+    path('get-conversion-rate/<str:query>', currency_views.get_currency_exchange_rate, name="get-conversion-rate")
```

Comment:
Change path name to `get-currency-conversion-rate`

take two inputs `source_currency_code` and `target_currency_code`